media/js/firefox/hello/start.js

```diff
@@ -195,6 +203,8 @@
     function bindHelloEvents() {
         Mozilla.UITour.observe(function(event, data) {
 
+            var tourSource = $main.data('tourSource');
```

Comment:
As this variable is used in a couple of different places/events, we could just read it once at the top of the JS file and use when needed to save repeated queries.